probe_scraper/transform_probes.py

```diff
@@ -182,10 +182,14 @@ def transform(probe_data, node_data, break_by_channel):
             print "  from: " + str({"node": node_id, "version": readable_version})
             for probe_type, probes in probe_data[channel][node_id].iteritems():
                 # Group the probes by the release channel, if requested
+                if probe_type not in result_data:
```

Comment:
The requirements from the [issue](https://github.com/mozilla/probe-scraper/issues/13#issuecomment-405236410) are a bit more involved than this: we should count, in this function, the number of probe types per channel.

Also, all the counts must be stored in `probes_counts`, not in `result_data`. The latter must not be modified.

Reply:
No. I'm getting this output only.

Reply:
The requirements from the issue are a bit more involved than this: we should count, in this function, the number of probe types per channel.
Also, all the counts must be stored in probes_counts, not in result_data. The latter must not be modified.

**OK. I will for that.

Reply:
Output was diffrent because I was checking that I have done correct or not. Sobefore returning I just print it(for checking).

Reply:
Issue #13 requires us to print the statistics as well, so let's print them at the end of the transformation, around the top level caller of this function.